doc/api/stream.md

```diff
@@ -369,12 +369,17 @@ See also: [`writable.uncork()`][].
 added: v8.0.0
 -->
 
-* `error` {Error}
+* `error` {Error} Optional, an error to emit with `'error'` event.
```

Comment:
```suggestion
* `error` {Error} Optional, an error used as an argument for any callbacks listening to the `'error'` event.
```

...or something like that? Nit-picky perhaps, but errors aren't emitted; events are. Anyway, optional nit, feel free to ignore.

Reply:
I thought I said that, and I don't want to reword the `EventEmitter.emit()` docs here. How about I reorder the words: "Optional, the `'error'` event will be emitted with the `error` as an argument."?

Reply:
That seems OK to me.

Reply:
(Although "as an argument for any listeners" or might be a hair better? Anyway, I'm OK with any of the possibilities here. Improvements/clarifications can always come at a later date.)